WordPress/proguard.cfg

```diff
@@ -72,6 +72,9 @@
 
 -keep class com.facebook.hermes.unicode.** { *; }
 -keep class com.facebook.jni.** { *; }
+
+-keep class com.swmansion.reanimated.** { *; }
+-keep class com.facebook.react.turbomodule.** { *; }
```

Comment:
These rules have been added following the installation guide of the Reanimated library:
https://docs.swmansion.com/react-native-reanimated/docs/fundamentals/installation/#proguard